### PR TITLE
Add homework 02.Git.Local file

### PR DESCRIPTION
**Pull request by iva-cpu**

### CODE REVIEW

**Comment by iva-cpu** [comment]
Add homework 03.Git.Hosting

File: .github/workflows/count.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Count tabs number in changed files
```
Comment (pluhin) [inline .github/workflows/count.yaml:1]: This file should not be here, as functional part current repo, please move to your report folder

**Review by pluhin — CHANGES_REQUESTED** [review]

File: .github/workflows/count.yaml (hunk 1 above)
Comment (pluhin) [reply .github/workflows/count.yaml:1]: I still see this file, please remove 
image

**Review by pluhin — CHANGES_REQUESTED** [review]

File: .github/workflows/count.yaml (hunk 1 above)
Comment (iva-cpu) [reply .github/workflows/count.yaml]: Already deleted it

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**